<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddRowDialog</class>
 <widget class="QDialog" name="AddRowDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>414</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:700;&quot;&gt;Please, fill this form for selling your bike.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Fields with * are necessary for filling.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="modelLabel">
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Model*</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="modelLineEdit">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="selling_priceLabel_2">
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Price*</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="priceLineEdit"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="yearLabel">
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Year</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="ownerLabel">
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Owner</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="comboBox">
     <property name="currentText">
      <string>1st owner</string>
     </property>
     <property name="maxVisibleItems">
      <number>3</number>
     </property>
     <item>
      <property name="text">
       <string>1st owner</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2nd owner</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3rd owner</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4th owner</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5th owner</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>6th owner</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>7th owner</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="mileageLabel">
     <property name="maximumSize">
      <size>
       <width>52</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Mileage*</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="mileagelineEdit"/>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="ex_priceLabel">
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Ex price</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="ex_priceLineEdit"/>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="gearboxLabel">
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string>Gearbox</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QComboBox" name="comboBox_2">
     <item>
      <property name="text">
       <string>Automatic</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1-speed</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2-speed</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3-speed</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4-speed</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5-speed</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>6-speed</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="QPushButton" name="applyButton">
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
   </item>
   <item row="9" column="2">
    <widget class="QPushButton" name="discardButton">
     <property name="text">
      <string>Discard</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <widget class="QSpinBox" name="spinBox">
     <property name="minimum">
      <number>1980</number>
     </property>
     <property name="maximum">
      <number>2022</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
